### PR TITLE
cleanup and setup docker compose

### PR DESCRIPTION
**Pull request by cosmingamanusi**

### CODE REVIEW

File: syncer/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,44 @@
-FROM rust:latest as builder
+# Step 1: Build the Rust project
+FROM rust:1.80.1-bookworm as rust-builder
 
-ENV DEBIAN_FRONTEND=noninteractive
+WORKDIR /usr/workspace
 
-RUN apt-get update && \
-    apt-get install -y wget curl gnupg software-properties-common lsb-release clang libclang-dev maven
+RUN git clone "https://github.com/bwarelabs/solana-syncer.git" -b "cleanup"
```
Comment (cosmingamanusi) [inline syncer/Dockerfile:6]: don t forget to remove -b cleanup

File: hbase/init.sh [hunk 2]
```diff
@@ -22,6 +22,10 @@ if [ ! -f table.tx-by-addr ]; then
   echo "create 'tx-by-addr', 'x'" | hbase shell
   touch table.tx-by-addr
 fi
+if [ ! -f table.tx_full ]; then
```
Comment (stefancristianco) [inline hbase/init.sh:25]: some name misalignment here tx_full or tx-by-block?

Comment (cosmingamanusi) [reply hbase/init.sh:25]: good catch, ty

File: docker-compose.syncer-from-geyser-plugin-to-COS.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+services:
+  syncer-local-files:
```
[the remark this answers is not in the document]
Comment (cosmingamanusi) [reply docker-compose.syncer-from-geyser-plugin-to-COS.yml:2]: done

**Pull request merged**